config/manager/kustomization.yaml

```diff
@@ -19,4 +19,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/nearnodeflash/nnf-dm
-  newTag: 0.0.1
+  newTag: b85a1e2070356e1083908cb7d4daf5957329fc65
```

Comment:
What exactly is the deal with this file? do we want to check in these updates?

Reply:
The "bin/kustomize edit" editing that file.  Usually you see this in the makefile, but Nate has moved it to the deploy.sh script.  Then "bin/kustomize build" is using that value to build the config of CRDs and resources.   That particular value is substituted here: https://github.com/NearNodeFlash/nnf-dm/blob/master/config/manager/manager.yaml#L33

We don't need to check  these in when they are private builds.